lib/ble.toit

```diff
@@ -4,60 +4,62 @@
 
 import binary
 import monitor
-import uuid as uuid_pkg
+import uuid
 import bytes
 import monitor show ResourceState_
-
-/**
-The base BLE UUID form which 16-bit and 32-bit UUIDs are generated.
-*/
-UUID/uuid_pkg.Uuid ::= uuid_pkg.parse "00000000-0000-1000-8000-00805F9B34FB"
-
-/**
-Returns the fully formed 128-bit uuid from a 16-bit or 32-bit BLE uuid.
-*/
-uuid value/int -> uuid_pkg.Uuid:
-  bytes := UUID.to_byte_array.copy
-  binary.BIG_ENDIAN.put_uint32 bytes 0 value
-  return uuid_pkg.Uuid bytes
-
-/**
-A 48-bit BLE advertise address.
-*/
-class Address:
-  raw_/ByteArray
-
-  constructor .raw_:
+import encoding.hex
+import device
+
+class BleUUID:
+  data_/any
```

Comment:
It might. The two different implementations (NimBLE and macos) uses different representations, so I kept it native to the representation to save converting back and forth when calling primitives.